zconfig.go

```diff
@@ -11,6 +15,7 @@ func init() {
 	DefaultRepository.AddProviders(Args, Env)
 	DefaultRepository.AddParsers(ParseString)
 	DefaultProcessor.AddHooks(DefaultRepository.Hook, Initialize)
+	DefaultProcessor.AddHooksEx(InitializeEx)
 }
 
 // Configure a service using the default processor.
```

Comment:
Add a ConfigureEx?

Reply:
Indeed, I missed it. It's not used in our code base.